src/main/java/org/rumbledb/api/Item.java

```diff
@@ -261,6 +261,24 @@ public DateTime getDateTimeValue() {
         throw new RuntimeException("Item is not a DateTime.");
     }
 
+    /**
+     * Returns the dateTime value of the item, if it is a atomic item of type dateItem.
+     *
+     * @return the dateTime value as a DateTime.
+     */
+    public DateTime getDateValue() {
+        throw new RuntimeException("Item is not a Date.");
+    }
+
+    /**
+     * Returns the dateTime value of the item, if it is a atomic item of type timeItem.
+     *
+     * @return the dateTime value as a DateTime.
+     */
+    public DateTime getTimeValue() {
+        throw new RuntimeException("Item is not a Time.");
+    }
+
```

Comment:
Do we need two methods? Could there be a single method implemented by all items that can return their value as a dateTime?

Reply:
Absolutely! I replaced getDateValue and getTimeValue with getDateTimeValue().